manual/manual/macros.hva

```diff
@@ -37,6 +37,12 @@
   text-align: center;
 }
 
+\newstyle{@media all}{
```

Comment:
Add a comment to explain why you do this. Are you sure it's a stripping issue, I would rather suspect a CSS order issue but I didn't see the generated css (if that's the case you could try a `!important`).

Reply:
It would be neat if, instead, we just had `<code>` blocks. But whatever works for now.

Reply:
Yeah good point. It's definitely something like stripping; putting just `\newstyle{.c003}{ font-size: XXX }` doesn't result in anything in the generated css

Reply:
Then again, targeting generated class names is probably not best practice: what if the number changes?

Reply:
That has bothered me a lot. I'd like the class names to be meaningful and retained. BTW, Hevea is now on Github: https://github.com/maranget/hevea